### ISSUE TITLE
Where is the actual source code?

### ISSUE THREAD

**Issue by racerxdl** [OpenScan-org/OpenScanCloud#7]
Where we can see the source code for the project?

**Comment by Jajaho** [comment]
AFAIK only the uploader is open source: [/main/uploader](https://github.com/OpenScan-org/OpenScanCloud/tree/main/uploader)

**Comment by racerxdl** [comment]
So `OpenScanCloud` not so open 😆

**Comment by Jajaho** [comment]
I'm afraid not when it comes to the source, but I do have to give Thomas credit. 
So far he has reliably provided it for free and vows to continue to do so, the operation is financed from donations.

I feel your point though, maybe he would like to chime in @OpenScanEu.

**Comment by RagingRoosevelt** [comment]
Addressed in this blog post: https://openscan.eu/blogs/news/is-openscan-an-open-source-project

**Comment by OpenScanEu** [comment]
@Jajaho @racerxdl True, the underlying code is not open and we do not plan to release it as detailed in the blog post linked by @RagingRoosevelt . It would not make any sense anyway, as the current solution requires dedicated hardware and setup.

BUT we plan on rebuilding the cloud functionality (started but currently paused [OpenScanCloud2](https://github.com/OpenScan-org/OpenScanCloud2) ) and we also want to open-source the local photogrammetry workers code, so that people can use their local photogrammetry CLI. Unfortunately, I can not give an exact timeline as we are currently focused on OpenScan3 (firmware). Though I hope to continue the work on the cloud solution somewhat at the end of this year/early next year)
